### ISSUE TITLE
Got a dead link when accidently disconnected and tried to reconnect.

### ISSUE THREAD

**Issue by sylwiabr** [enso-org/enso#6725]
### What type of issue is this?

Transient – Occurring only once

### Is this issue blocking you from using Enso?

- [ ] Yes, I can't use Enso because of this issue.

### Is this a regression?

- [ ] Yes, previous version of Enso did not have this issue.

### What issue are you facing?

https://github.com/enso-org/enso/assets/12892578/036955ff-0887-4a30-8785-e9ee239b7cf2

Unfortunately I don't have the logs but opening the issue for tracking purposes.

### Expected behaviour

Proper connection between nodes,

### Enso Version

develop 17.05

### Browser or standalone distribution

Standalone distribution

### Browser Version or standalone distribution

standalone

### Operating System

Windows

**Comment by Akirathan** [comment]
Seems like a duplicate of #6772? cc @kazcw

**Comment by kazcw** [comment]
Yes, this is #6772.

**Comment by kazcw** [comment]
(duplicate)

**Comment by kazcw** [comment]
(Close as "Not planned" wasn't working.)
